management_api_app/models/domain/resource.py

```diff
@@ -18,7 +17,10 @@ class ResourceType(str, Enum):
 
 
 class Resource(AzureTREModel):
-    id: UUID
-    description: str
+    id: str
+    resource_name: str
+    resource_version: str
+    resource_parameters: dict
     resourceType: ResourceType
     status: Status
+    isDeleted: bool = False
```

Comment:
Shouldn't this be `is_deleted` to match the convention?

Reply:
Ok. I'm fine with either way as long as *all* the variable names follow the same convention. `resource_name` and `resource_version` vs `resourceType` and `isDeleted`.

Reply:
Updated so that it is consistently using camelCase (per teams discussion)